### PR TITLE
CI: build_report job no dependcies on yml

### PR DESCRIPTION
**Pull request by maxknv**
#no_merge_comit
 #job_ClickHouse_build_check
 #job_ClickHouse_special_build_check
 #job_style_check

### Changelog category (leave one):
- CI Fix or Improvement (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
fix build_report job so that it's defined by ci_config only (not yml file)

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

**Comment by robot-clickhouse-ci-1** [comment]
*This is an automated comment for commit 108eac1df2072b4950f26029ba5c0c583793124e with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/60613/108eac1df2072b4950f26029ba5c0c583793124e/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

**Comment by maxknv** [comment]
one more pr to unblock "abstract yml wf" https://github.com/ClickHouse/ClickHouse/pull/60421

File: tests/ci/pr_info.py [hunk 1]
```diff
@@ -288,6 +289,11 @@ def __init__(
     def is_master(self) -> bool:
         return self.number == 0 and self.head_ref == "master"
 
+    def is_release(self) -> bool:
+        return self.number == 0 and bool(
+            re.match(r"^2[1-9]\.[1-9][0-9]*$", self.head_ref)
```
Comment (Felixoid) [inline tests/ci/pr_info.py:294]: Let's use `'release' in self.labels`. It will cover everything, including cloud releases

Comment (maxknv) [reply tests/ci/pr_info.py:294]: to be done

**Pull request merged**